examples/throughput_c/CMakeLists.txt

```diff
@@ -3,12 +3,11 @@ include($ENV{XMOS_CMAKE_PATH}/xcommon.cmake)
 project(fileio_test)
 
 # sandbox
-set(XMOS_SANDBOX_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../../..)
+set(XMOS_SANDBOX_DIR ${CMAKE_CURRENT_LIST_DIR}/../../..)
 
 # target ans srcs
 set(APP_HW_TARGET XCORE-AI-EXPLORER)
-set(APP_XC_SRCS main.xc)
-set(APP_C_SRCS test.c)
+set(APP_C_SRCS main.c)
```

Comment:
The source files and xscope.config file should be inside a `src` directory. Then this line and `set(APP_XSCOPE_SRCS config.xscope)` should be removed. This also applies to the other example.